cmd/olfullnode/genesis.go

```diff
@@ -76,6 +78,8 @@ func init() {
 	//genesisCmd.Flags().BoolVar(&genesisCmdArgs.allowSwap, "enable_swaps", false, "Allow swaps")
 	genesisCmd.Flags().IntVar(&genesisCmdArgs.numValidators, "validators", 4, "Number of validators to initialize mainnetnet with")
 	genesisCmd.Flags().IntVar(&genesisCmdArgs.numNonValidators, "nonvalidators", 1, "Number of fullnodes to initialize mainnetnet with")
+	genesisCmd.Flags().IntVar(&genesisCmdArgs.numWitness, "witness", 0, "number of witness to initialize mainnet with")
```

Comment:
I want to call it ethereum withness, instead of witness here... Just calling it witness, we are adding more work when other chains are integrated..